### DISCUSSION TITLE
Sesion

### DISCUSSION THREAD

**Discussion by oscardanielM** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#19] [Q&A]
Hola, alguien sabe si hoy hay sesión de live 😥

**Comment by GloriaCoralCerecedo** [comment]
Hola, solo Lunes, Miércoles y Viernes

**Reply by oscardanielM** [reply]
Tendrian el link de sesion de hoy?

**Reply by AlexAntonioG** [reply]
Corrección, las sesiones son lunes, miércoles y jueves

**Reply by soygussil** [reply]
según yo son lunes, miercoles y jueves

**Reply by GloriaCoralCerecedo** [reply]
Así me confundí, disculpen :D
